apps/api/src/domain/models/song/commands/song-created.event.ts

```diff
@@ -12,7 +13,7 @@ export type EventRecordMetadata = {
  * For now, we are just mirroring our command payload onto our events. We can update
  * this if and when we decide to use the events (e.g. for event sourcing).
  */
-export class SongCreated extends BaseDomainModel {
+export class SongCreated extends BaseDomainModel implements IEvent {
```

Comment:
Can we make sure we explicitly implement `IEvent` in all event classes?

Reply:
Actually, I went with extending a `BaseEvent` class, so this isn't an issue any more.